pkg/cmd/generate-pg-catalog/main.go

```diff
@@ -34,8 +34,9 @@ import (
 const getServerVersion = `SELECT current_setting('server_version');`
 
 var (
-	postgresAddr = flag.String("addr", "localhost:5432", "Postgres server address")
-	postgresUser = flag.String("user", "postgres", "Postgres user")
+	postgresAddr    = flag.String("addr", "localhost:5432", "Postgres server address")
+	postgresUser    = flag.String("user", "postgres", "Postgres user")
+	postgresCatalog = flag.String("catalog", "pg_catalog", "Catalog or namespace, default: pg_catalog")
```

Comment:
My last comment is to rename these variables here to be more generic since it's no longer specific to pg_catalog anymore.

Also we should rename the directory as well since generate-pg-catalog is no longer accurate.

Reply:
These variables are specific for connecting to postgres

Reply:
maybe instead of the directory `generate-pg-catalog` we can name it `generate-postgres-metadata-tables`

and instead of the variable `postgresCatalog` it can be called `postgresSchema`

Reply:
Done.